src/ontology/cl-edit.owl

```diff
@@ -21531,6 +21535,7 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "CL:CVS"^^xsd:string) obo:IAO_
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0005010 "FMA:86560"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0005010 "intercalated cell"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0005010 "cell"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25632105"^^xsd:string) rdfs:comment obo:CL_0005010 "Intercalated cells are also called “mitochondria-rich cells,” reflecting the high levels of round mitochondria at their apical pole, or in their cytoplasm, a distribution that was quite different from that of other kidney tubule cells, which accumulated mitochondria around the basolateral membrane. These cells also have “numerous irregular apical microvilli” compared with the surrounding segment-specific cells. Intercalated cells lack a central cilium, at least in the cortex, which also differentiates them from the adjacent principal cells.")
```

Comment:
“mitochondria-rich cells,” => “mitochondria-rich cells”,
a distribution that was quite different => ...that is...
These cells also have “numerous irregular apical microvilli” => I'd remove the quotation marks

Reply:
Thank you for the review, @paolaroncaglia!
Edits made.
Of note, I believe the standard for this ontology is to use the American spelling variants. Does that standard extend to punctuation as well? For your first suggestion, I believe the American standard places the comma inside the quotation marks and the British standard would place it outside. I selected the latter as per your suggestion.

Reply:
@bvarner-ebi yes, standard for GO and some other ontologies for sure is to use US spelling. I've not been advised on punctuation specifically, but TBH, I've never come across punctuation inside quotation marks in the 4 ontologies I've worked on. English is not my first language, but my first language follows the same standard as the British one - commas inside quotation marks would be considered a (minor) error. :-)